wongnung/models.py

```diff
@@ -101,9 +101,21 @@ def get_film(cls, film_id: str) -> Film:
             response_info = tmdb.Movies(num_id).info()
             response_credits = tmdb.Movies(num_id).credits()
             title = response_info["title"]
-            year_released = response_info["release_date"].split("-")[0]
+
             summary = response_info["overview"]
-            poster = f"https://image.tmdb.org/t/p/w600_and_h900_bestv2{response_info['poster_path']}"
+            if summary == '':
```

Comment:
Use `not` for empty strings instead
```suggestion
            if not summary:
```

Reply:
Fixed.